### DISCUSSION TITLE
Which video players are compatible with react-native-web?

### DISCUSSION THREAD

**Discussion by rcassity** [necolas/react-native-web#2070] [Q&A] [answered]
I am using react-native-web and I need a video player that is compatible with react-native-web. Videojs has monkey patches for react-native-web and I am not sure that Videojs is the best video player. What have you used for playing videos? Has anyone used react-native-web-video?

Thanks!

**Comment by necolas** [comment, marked as answer]
You can use any DOM or React DOM-compatible video player. For some reason, Videojs is modifying the native event object, which will make it conflict with frameworks that also do that to emulate React's synthetic event interface.

**Reply by rcassity** [reply]
Thank you for your answer. I will look into those.
